server/src/packages/pas-form/applicants/applicants.controller.ts

```diff
@@ -15,15 +32,16 @@ export const APPLICANT_ATTRIBUTES = [
   'dcp_state',
   'dcp_zipcode',
   'dcp_phone',
-  'dcp_type',
+  'dcp_type', // only on dcp_applicantinformation entity, not applicantrepresentative
+  'targetEntity', // custom attribute to route to/from correct crm entity
```

Comment:
Originally, the `APPLICANT_ATTRIBUTES` were supposed to be a list of valid CRM attributes names _including our internal naming for relationships (`dcp_project` is `projects`)_.

Now that `targetEntity` is included, it will be allowed through the `PATCH` and `POST` methods, and CRM will reject it because it's not a real key. We might want to keep this "one-off" handling in the controller methods themselves until we're comfortable handling this sort of situation in a more generic way.

Reply:
I see, instead am going to organize it like this, adding a second ATTRIBUTES variable to show the different attributes for the applicant vs. appicant rep entities.   Trying to use `targetEntity` in the http methods instead:
```js
export const APPLICANT_REPRESENTATIVE_ATTRIBUTES = [
  'dcp_firstname',
  'dcp_lastname',
  'dcp_organization',
  'dcp_email',
  'dcp_address',
  'dcp_city',
  'dcp_state',
  'dcp_zipcode',
  'dcp_phone',
];

export const APPLICANT_ATTRIBUTES = [
  ...APPLICANT_REPRESENTATIVE_ATTRIBUTES,
  'dcp_type',
];
```

Reply:
@allthesignals thanks for flagging this -- super helpful.

Currently how I addressed this in POST and PATCH is filter to remove `target_entity` from the `allowedAttrs,`
```js
      allowedAttrs = pick(body, APPLICANT_REPRESENTATIVE_ATTRIBUTES.filter(attribute => attribute !== 'target_entity'));
```
This is a little duplicative, but it works for now.  Happy to adjust to make it more ergonomic - let me know if you prefer it another way.

Reply:
The reason I kept it in this array, is that I needed it in packages controller to be able to pass that information along to the frontend `applicants` array so the frontend can know to render as `Other Team Member`